pgr-services/src/main/java/org/egov/pgr/service/MigrationService.java

```diff
@@ -189,22 +221,28 @@ private void setStatusToUUIDMap(){
         String serviceCode = serviceV1.getServiceCode();
         String serviceRequestId = serviceV1.getServiceRequestId();
         String description = serviceV1.getDescription();
-        String source = serviceV1.getSource().toString();
+        String source;
+        if(!ObjectUtils.isEmpty(serviceV1.getSource()))
+            source = serviceV1.getSource().toString();
+        else
+            source = "web";
```

Comment:
Why default value is web?

Reply:
When serviceV1.getSource() is null, should I let the source String have the value null?